cli/pkg/local/app.go

```diff
@@ -146,6 +146,13 @@ func NewApp(ctx context.Context, ver config.Version, verbose bool, olapDriver, o
 		return nil, err
 	}
 
+	if reset {
+		err := drivers.Drop(olapDriver, map[string]any{"dsn": olapDSN}, logger)
+		if err != nil {
+			return nil, fmt.Errorf("failed to clean OLAP: %w", err)
+		}
+	}
```

Comment:
1. This should run before creating the instance (around line 122)
2. Put the olapConfig into a new variable and use it here and on line 140 to avoid duplication